web-ui/src/components/reviews/TeamReviews.jsx

```diff
@@ -260,12 +261,13 @@ const TeamReviews = ({ periodId }) => {
           : null;
       if (data && data.length > 0) {
         data = data.filter((review)=>"canceled".toUpperCase() !== review?.status?.toUpperCase());
-        newSelfReviews[teamMember.id] = data[0];
+        newSelfReviews[data[0].requesteeId] = data[0];
```

Comment:
Is it okay to assume we just want the first team member?
Is there a possibility that `data` is an empty array?

Reply:
There's an existing check for `data` being empty a couple of lines up. That said...wow...bug miss on my part. I'm not sure where my head was on this one. Fixing now.